resources/assets/js/components/TopBar.vue

```diff
@@ -167,8 +167,8 @@ Note that the page editing toolbar is a separate component found in `components/
 			backToSites() {
 				// another prompt to save the page when going back to the site listing
 				this.promptToSave(() => {
-					this.$store.commit('setPage', {});
 					this.$store.commit('setLoaded', false);
+					this.$store.commit('setPage', {});
```

Comment:
Changing the order of this seemed to fix an intermittent issue I was having where the back to sites button stopped working as somewhere the code was trying to access page data (I think in updating the site name and url in the title bar).